ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Envelope.scala

```diff
@@ -44,7 +45,7 @@ object Envelope {
 
     private val Prefix = Envelope.Prefix :+ "Throughput"
 
-    val All =
+    val All: immutable.Seq[Envelope] =
```

Comment:
I would have preferred just using `Vector` everywhere.

Reply:
Fair. I just went with what IntelliJ recommended and didn't think too hard about it.